tests/_support/index.js

```diff
@@ -77,6 +77,7 @@ Cypress.Commands.add("configureCluster", function(configuration) {
 
     router
       .route(/service\/marathon\/v2\/apps/, "fx:marathon-1-task/app")
+      .route(/net\/v1\/nodes/, "fx:1-app-for-each-health/nodes")
```

Comment:
Yes that is true as far as i remember the Mesos Stream does not work correctly on cypress and since Node informations are from there I didn't want to write a test. But I could try to since now the integration test tooling is kinda working.

Reply:
Also the nodes we have in our fixtures do have actual hostnames and we can't match them anyways. So the only thing we could check is if the N/A is set.